test/libsolidity/semanticTests/functionCall/precompile_extcodesize_check.sol

```diff
@@ -0,0 +1,24 @@
+interface Identity {
+    function selectorAndAppendValue(uint value) external pure returns (uint);
+}
+contract C {
+    Identity constant i = Identity(address(0x0004));
+    function testHighLevel() external pure returns (bool) {
+        // Should fail because `extcodesize(4) = 0`
+        i.selectorAndAppendValue(5);
+        return true;
+    }
+    function testLowLevel() external view returns (uint decoded) {
+        (bool success, bytes memory ret) =
+            address(4).staticcall(
+                abi.encodeWithSelector(Identity.selectorAndAppendValue.selector, uint(5))
+            );
+        decoded = abi.decode(ret, (uint));
```

Comment:
We need something like `abi.decodePacked` to properly decode this. I think the return value is [4-byte selector, 32-byte padded value 5]`.